dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```
@@ -2030,10 +2042,10 @@
       "type": "object",
       "description": "Supported parameters for a Conversational PII detection and redaction task.",
       "properties": {
-        "piiCategories": {
+        "inclusionList": {
           "$ref": "#/definitions/ConversationPIICategories"
         },
-        "includeAudioRedaction": {
+        "includeAudioRedactionTiming": {
           "type": "boolean",
           "description": "Flag to indicate if audio redaction is requested. By default audio redaction will not be performed.",
```

Consider adjusting the docstring as well for clarity. I would suggest:
```suggestion
          "description": "Flag to indicate if response should include audio stream offset and duration for any detected entities to be redacted. By default, audio timing of redacted entities are not included.",
```